lib/still/compiler/incremental/node.ex

```diff
@@ -64,13 +65,17 @@ defmodule Still.Compiler.Incremental.Node do
 
   @impl true
   def handle_call(:compile, _from, state) do
-    with result <- Compile.run(state) do
+    with {:ok, source_file} <- Compile.run(state) do
       BrowserSubscriptions.notify()
-      {:reply, result, state}
+      FileErrorsCache.set({:ok, source_file})
```

Comment:
How about we name this `ErrorBus` or `ErrorQueue` or `ErrorCache`. Doesn't need to be file errors necessarily right? What if there's an Elixir compilation error, will it show up here?

Reply:
It was called `FileErrorsBus`before, but I didn't like it. I like `ErrorCache`, but I also like `SourceFileErrors`. I think this will be heavily coupled with errors related to the compilation of source files. What do you think?

Reply:
I prefer `ErrorCache`, I think it would be better in the codebase. Especially because there aren't a lot of other errors that we might handle. If you think we should be *even more* specific, then either `CompilationErrorCache` or `SourceFileErrors` sounds good 👍 

Really, any of these options is good, go with your gut 😄

Reply:
👍